src/org/zaproxy/zap/extension/pscanrules/TestInfoPrivateAddressDisclosure.java

```diff
@@ -1,153 +1,159 @@
-/*

```

Comment:
Something is changing the line endings then (git? EGit?), worth checking the git configs in your local repo.